server/server.go

```diff
@@ -161,44 +166,58 @@ const (
 	ExtensionsPath = "/pd/apis"
 )
 
-func combineBuilderServerHTTPService(svr *Server, apiBuilders ...HandlerBuilder) (http.Handler, error) {
-	engine := negroni.New()
+func combineBuilderServerHTTPService(svr *Server, serviceBuilders ...HandlerBuilder) (map[string]http.Handler, error) {
+	userHandlers := make(map[string]http.Handler)
+	registerMap := make(map[string]struct{})
+
+	apiHandler := negroni.New()
```

Comment:
`apiHandle`r may not be a good naming

Reply:
In the original code `apiHandler, err := combineBuilderServerHTTPService(s, apiBuilders...)` it was called apiHandler. I think it would be even more confusing if we use engine as the name here.